src/components/statsBar/AllStatsController.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
You don't need these controllers!! Can you just update the view? As we don't want to depend on redux at two places